structuralalignment/superposition/theseus.py

```diff
@@ -1 +1,114 @@
-#here is the theseus code
+import atomium
+import subprocess
+
+# usage: Theseus(["3dk3"])
+class Theseus:
+    def __init__(self, pdbs):
+        self.pdbs = pdbs
+        self.fetched_pdbs = []
+        for pdb in pdbs:
+            self.fetched_pdbs.append(atomium.fetch(pdb))
+        self.align()
+
+    def align(self):
+        self.fetched_pdbs_filename = []
+        """
+        for m in range(len(self.fetched_pdbs)):
+            self.fetched_pdbs[m].model.save(self.pdbs[m])
+        """
+        for m in range(len(self.fetched_pdbs)):
+            self.fetched_pdbs[m].model.save(
+                self.pdbs[m] + "." + self.fetched_pdbs[m].filetype
+            )
+            self.fetched_pdbs_filename.append(
+                self.pdbs[m] + "." + self.fetched_pdbs[m].filetype
+            )
+
+        output = subprocess.check_output(["theseus",] + self.fetched_pdbs_filename)
+        return self._parse(output)
+
+    def _parse(self, output):
+        print(output)
+
+
+
+
+#usage: TheseusAlign(["3DK3"])
+class TheseusAlign:
+    def __init__(self, pdbs):
+        self.pdbs = pdbs
+        self.fetched_pdbs = []
+        self.fetched_pdbs_filename = []
+
+        self.fastafile = "theseus.fasta"
+        self.filemap_file = "theseus.filemap"
+        self.alignment_file = "theseus.aln"
+        self.alignment_app = "/usr/local/bin/muscle"
```

Comment:
We shouldn't rely on system packages. This will be provided by `conda`, so we just trust it will be somewhere in `$PATH`. Thus, `muscle` is enough (no abs paths).